src/test/scala/chisel3/iotesters/ChiselPokeSpec.scala

```diff
@@ -0,0 +1,40 @@
+// See LICENSE for license details.
+
+import chisel3._
+import chisel3.iotesters.experimental.{ChiselPokeSpec, ChiselImplicitPokeSpec}
+
+class MyDut extends Module {
+  val io = IO(new Bundle {
+    val in = Input(UInt(8.W))
+    val out = Output(UInt(8.W))
+  })
+  io.out := io.in + 1.U
+}
+
+class MyDutSpec extends ChiselPokeSpec {
```

Comment:
It looks like ChiselPokeSpec is based on FlatSpec, I think you should consider FreeSpec
I think FreeSpec gives writer ability to write more expressive test harnesses